knoten/utils.py

```diff
@@ -1,4 +1,296 @@
 import pyproj
+import numpy as np 
+from typing import NamedTuple
+
+class Point(NamedTuple):
+    x: np.double
+    y: np.double
+    z: np.double
+
+class LatLon(NamedTuple):
+    lat: np.double
+    lon: np.double
+
+class Sphere(NamedTuple):
+    lat: np.double
+    lon: np.double
+    radius: np.double
+
+class Matrix(NamedTuple):
+    vec_a: Point
+    vec_b: Point
+    vec_c: Point
```

Comment:
@Kelvinrr Is this what you had in mind or did you mean forgoing the classes altogether?